infra/src/main/resources/i18n/translations.fi.json

```diff
@@ -40,6 +40,7 @@
                 "VerticalCoordinateSystem": "Korkeusjärjestelmän määritys on virheellinen",
                 "Code": "Kentän sisältö {{value}} on virheellinen",
                 "FreeText": "Tekstikentän sisältö {{value}} on virheellinen",
+                "FreeTextWithNewLines": "Tekstikentän sisältö {{value}} on virheellinen",
```

Comment:
Tämä puuttui (esim. IM-tiedoston huomiokentät on näitä)

Reply:
Kappas, muistan kun muutin tuon sallimaan uudet rivit. Ei tullut silloin pieneen mieleenkään että virheviesti olisi uupunut 🤔 

Vaikea kyllä sanoa miten tällaiset jäisivät herkemmin kiinni